resource/css/skosmos.css

```
@@ -951,3 +954,82 @@ body {
     font-size: 20px !important;
   }
 
+/* Tooltips */
+
+.tooltip-html {
+  position: relative;
+  display: inline-block;
+  cursor: pointer;
+}
+
+#concept-label .tooltip-html {
+  bottom: 4px;
+}
+
+.tooltip-html button {
+  line-height: 1.0;
+  padding: 3px;
+  margin: -3px;
+}
+
+.tooltip-html button:focus {
+  border: 2px solid var(--vocab-text);
+  border-radius: 0;
```

I would add a compensation for the border in the margin like so:

```suggestion
  border-radius: 0;
  margin: -4px;
```
So that the text doesn't jitter when the button is pushed (Chromium, not sure how it would be with other browsers).